scripts/serving/wrapper/preprocessor.py

```diff
@@ -78,10 +106,13 @@ def start(self):
 
         # Phase 1: Extract model_format and yaml
         yaml_data = self._extract_yaml()
-        format = yaml_data["format"].lower()
+        format = yaml_data["format"]
 
-        # Phase 2: Generate 'config.pbtxt'
-        self._generate_config_pbtxt(yaml_data, self._serving_name)
+        # Phase 2: Generate 'config.pbtxt' if need
+        if format != 'PMML' and format != 'SKLearn' and format != 'XGBoost':
+           self._generate_config_pbtxt(yaml_data, self._serving_name)
+
+        self._generate_model_setting(format)
```

Comment:
Having any condition to run `self._generate_model_setting(format)`?

Reply:
The condition judge is in the function `self._generate_model_setting(format)`.